localisation/00_wc_artifact_localisation.csv

```diff
@@ -1,22 +1,29 @@
+atiesh_desc;Atiesh, the greatstaff of the Guardian is a powerful staff passed down through the line of the Guardians of Tirisfal. The staff was born from the �seed of hate� and grants its wielder �unending power�.;;Atiesh, der Langstab des W�chter ist ein m�chtiger Stab, der von einem W�chter von Tirisfal an den n�chsten weitergegeben wird. Der Stab wurde aus dem �Samen des Hasses� geboren und gew�hrt seinem Besitzer �unendliche Macht�.;;;;;;;;;;;x
 book_of_medivh;Book of Medivh;Livre de Medivh;Medivhs Zauberbuch;;;;;;;;;;;x
-book_of_medivh_desc;The spellbook written by Medivh.;;;;;;;;;;;;;x
+book_of_medivh_desc;The spellbook written by Medivh.;Le livre envo�tant �crit par Medivh.;Das Zauberbuch von Medivh.;;;;;;;;;;;x
 dark_crystal;Dark Crystal;;Dunkler Kristall;;;;;;;;;;;x
-dark_crystal_desc;Void magic emanates from this strange crystal, but somewhere in the depths of it you feel the presence of the Light.;;Aus diesem seltsamen Kristall str�mt die Magie der Leere hervor, aber irgendwo in der Tiefe sp�rt man die Gegenwart des Lichts.;;;;;;;;;;;x
+dark_crystal_desc;Void magic emanates from this strange crystal, but somewhere in the depths of it you feel the presence of the Light.;;Aus diesem seltsamen Kristall entspringt die Magie der Leere, aber irgendwo in seinem Inneren sp�rt Ihr die Pr�senz des Lichts.;;;;;;;;;;;x
 doomhammer;Doomhammer;;Schicksalshammer;;;;;;;;;;;x
-doomhammer_desc;The Doomhammer was forged in a basin of pristine lava on Draenor and passed from father to son for decades. Despite its oversized dimensions, this hammer absorbed the strength of elementals and is capable of inflicting a crushing blow to any enemy.;;;;;;;;;;;;;x
+doomhammer_desc;Forged on Draenor in a lava basin and inherited from father to son, this mighty hammer absorbs the power of the elements and is capable of dealing a devastating blow to any enemy.;;Der Schicksalshammer wurde auf Draenor in einem Lava-Becken geschmiedet und vom Vater an den Sohn vererbt.Dieser gewaltige Hammer absorbiert er die Kraft der Elemente und ist in der Lage, jedem Gegner einen vernichtenden Schlag zu versetzen.;;;;;;;;;;;x
 eye_of_sargeras;Eye of Sargeras;;Auge von Sargeras;;;;;;;;;;;x
+eye_of_sargeras_desc;The Eye of Sargeras is a powerful gem-like artifact previously used by the Avatar of Sargeras.;;Das Auge des Sargeras ist ein m�chtiges, edelstein�hnliches Artefakt, das zuvor vom Avatar des Sargeras benutzt wurde.;;;;;;;;;;;x
+felomelorn;Felo'melorn;;;;;;;;;;;;;x
+felomelorn_desc;A family relic of the Sunstrider dynasty. Anastarian Sunstrider wielded it against the Amani trolls after the founding of Quel'thalas. It is said that a thousand trolls fell before Fel'melorn's fury.;;Eine Familienreliquie der Sunstrider-Dynastie. Anastarian Sunstrider schwang es gegen die Amani-Trolle nach der Gr�ndung von Quel'thalas. Es wird gesagt, dass tausend Trolle Felo'melorns Wut zum Opfer fielen.;;;;;;;;;;;x
```

Comment:
```suggestion
felomelorn_desc;This runeblade, known also as Flamestrike, is a family relic of the Sunstrider dynasty. Anasterian Sunstrider wielded it against the Amani trolls after the founding of Quel'thalas, having originally inherited it from Dath'Remar. It is said that a thousand trolls fell before Felo'melorn's fury.;;Eine Familienreliquie der Sunstrider-Dynastie. Anastarian Sunstrider schwang es gegen die Amani-Trolle nach der Gr�ndung von Quel'thalas. Es wird gesagt, dass tausend Trolle Felo'melorns Wut zum Opfer fielen.;;;;;;;;;;;x
```

Reply:
done, but deleted last sentence about trolls because of length